### PR TITLE
[l10n_cn] : China Accounting Localization working out of the box

### PR DESCRIPTION
**Pull request by rop-odoo**
Task: https://www.odoo.com/web?#id=34299&view_type=form&model=project.task&action=327&menu_id=4720
Pad:  https://pad.odoo.com/p/r.cbb16eec8e23668032afd6f0e7fe2ba5

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_cn/data/account_tax_group_data.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="l10n_cn_tax_group_gst_3" model="account.tax.group">
+        <field name="name">GST 3%</field>
```
Comment (elicoidal) [inline addons/l10n_cn/data/account_tax_group_data.xml]: Better VAT than GST

**Comment by tarbib** [comment]
Hello

Is it possible to have access to this task ?
https://www.odoo.com/web?#id=34299&view_type=form&model=project.task&action=327&menu_id=4720

File: addons/l10n_cn_standard/__manifest__.py [hunk 2]
```diff
@@ -35,4 +35,5 @@
         'data/account_tax_template_data.xml',
         'data/account_chart_template_data.xml',
     ],
+    'auto_install': True
```
Comment (Yenthe666) [inline addons/l10n_cn_standard/__manifest__.py]: Uhh, are we sure here?

**Comment by jeffery9** [comment]
10_cn should be installed auotomaticlly，but not right for the other two.

发自我的华为手机

-------- 原始邮件 --------
主题：Re: [odoo/odoo] [l10n_cn] : China Accounting Localization working out of the box (#21169)
发件人："Yenthe V.G"
收件人：odoo/odoo
抄送：Subscribed


@Yenthe666 commented on this pull request.

________________________________

In addons/l10n_cn_standard/__manifest__.py<https://github.com/odoo/odoo/pull/21169#discussion_r157192150>:

> @@ -35,4 +35,5 @@
         'data/account_tax_template_data.xml',
         'data/account_chart_template_data.xml',
     ],
+    'auto_install': True


Uhh, are we sure here?

―

File: addons/l10n_cn/__init__.py [hunk 3]
```diff
@@ -2,3 +2,15 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 # Copyright (C) 2007-2014 Jeff Wang(<http://jeff@osbzr.com>).
+
+from odoo import api, SUPERUSER_ID
+
+
+def _auto_install_l10n_cn(cr, registry):
```
Comment (qdp-odoo) [inline addons/l10n_cn/__init__.py]: useless post_init_hook: to remove

Comment (rop-odoo) [reply addons/l10n_cn/__init__.py]: @qdp-odoo 
> When I create a database and give the country 'China'  all the COA are installed.
- here is the case that for I need to use post_init_hook
> when I manually select the COA then It should only install the selected COA not all.
 - If I install l10n_cn => It should automatically install l10n_cn + l10n_cn_small_business
- If I install l10n_cn_small_business => It should automatically install  l10n_cn + l10n_cn_small_business
- If I install l10n_standard => It should automatically install l10n_cn + l10n_cn_standard

**Comment by qdp-odoo** [comment]
landed in master, for saas 11.2, at revision https://github.com/odoo/odoo/commit/6d92fc93f616c7f943d0e9bee064272feebc6786

**Comment by jeffery9** [comment]
this is broken, please revert.

**Comment by mart-e** [comment]
@jeffery9 this commit has been merged months ago, it is too late to revert.
However, if you explain what is wrong, we can work to fix it.

**Comment by jeffery9** [comment]
The short-code cn means China, and tw means Taiwan, so  l10n_cn_small_business is used for China, not for China Taiwan, but the language used is traditional Chinese, not simplified Chinese.

The wrong language used is file  `addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_data.xml`

thanks

**Comment by rop-odoo** [comment]
@mart-e ,
When we started working on this task at that time there was another CSV file we had as a source(which was different from the latest one linked in related issue) and branch was created using the terms from that source file, so I think I might have forgotten to replace few terms from latest CSV when it was provided in the issue, because I just came to know there exists two versions of Chinese language from this task only. Initially, I think RDE provided the source in the task.

**Comment by mart-e** [comment]
@rop-odoo thanks for the info, I see @jeffery9 has fixed it at #27092, will check that one

**Pull request closed without merging**